### PR TITLE
return results instead of none when CODS soiling signal is small

### PR DESCRIPTION
**Pull request by martin-springer**
- [x] Code changes are covered by tests
- [x] Code changes have been evaluated for compatibility/integration with TrendAnalysis
- ~~[ ] New functions added to `__init__.py`~~
- ~~[ ] API.rst is up to date, along with other sphinx docs pages~~
- [x] Example notebooks are rerun and differences in results scrutinized
- [x] Updated changelog

### CODE REVIEW

**Comment by mdeceglie** [comment]
I re-ran the offending notebook and scrutinized the output to be sure no substantial changes occurred in the outputs. That plot does appear to change size, but I don't think it is a concern.

File: rdtools/test/soiling_cods_test.py [hunk 1]
```diff
@@ -111,13 +111,26 @@ def test_soiling_cods(cods_normalized_daily):
...
-        assert x in expected_summary_columns,\
+        assert x in expected_summary_columns, \
             "'{}' not an expected column in result_df]".format(x)
     for x in expected_summary_columns:
-        assert x in actual_summary_columns,\
+        assert x in actual_summary_columns, \
             "'{}' was expected as a column, but not in result_df".format(x)
 
 
+def test_soiling_cods_small_signal(cods_normalized_daily_small_soiling):
```
Comment (martin-springer) [inline rdtools/test/soiling_cods_test.py:121]: I added the CODS small signal test

**Comment by martin-springer** [comment]
I did update all packages in my conda environment and saved the notebook. Somehow, there's still a discrepancy in one cell. I could try re-creating a new environment the exact way nbval does here. On the other hand, we get quite a lot of warning and deprecation messages. Might be time to start resolving them.

**Comment by mdeceglie** [comment]
I think we can look past the notebook failure this time I opened #401 today because of the message we're seeing in the notebook.

**Comment by mdeceglie** [comment]
Let's get the change log updated, then I think we can move forward with this. The version will be `2.2.0-beta.2`

**Comment by martin-springer** [comment]
Sounds good, I updated the changelog. Could you also merge these changes into `aggregated_filters_for_trial` and create a tag for the next pvfleets run?

**Review by mdeceglie — APPROVED** [review]
Looks good. Thanks @martin-springer!

**Pull request merged**